### PR TITLE
Transfer Tracks list in the same order

### PR DESCRIPTION
**Pull request by MahdiAkrami01**
Reverse Tracks list And Send 1 by 1 to transfer them in the same order

### CODE REVIEW

File: webapp/app/services/TrackService.js [hunk 1]
```diff
@@ -23,11 +23,14 @@ angular.module('trikatuka2').service('TrackService', function (Spotify, $q, Requ
         }
 
         getAll(user).then(function(tracks){
-            var pages = Math.ceil(tracks.length / 50);
+            
+            tracks.reverse()
+            
+            var pages = Math.ceil(tracks.length / 1);
```
Comment (aartek) [inline webapp/app/services/TrackService.js:29]: Dividing by 1 makes no sense. Pagination is lost due to this change

File: webapp/app/services/TrackService.js [hunk 2]
```diff
@@ -23,11 +23,14 @@ angular.module('trikatuka2').service('TrackService', function (Spotify, $q, Requ
...
 
             var toTransfer = [];
             for(var i=0; i<pages; i++) {
-                var data  = tracks.slice(i * 50, (i * 50) + 50);
+                var data  = tracks.slice(i * 1, (i * 1) + 1);
```
Comment (aartek) [inline webapp/app/services/TrackService.js:33]: `i*1` is just an `i`. Also, it's inefficient to divide data into single items.

**Review by aartek — COMMENTED** [review]
Thank you for your contribution, but I cannot accept this pull request. I will work on this soon.

**Comment by AkramiPro** [comment]
I do this to save old pattern so you can compare changes easier . 
you should send tracks one by one to save the order . if you send bulk of tracks , spotify api change the order of array and save it in Alphabet order . 
and we don't need pagination in this service.

**Comment by AkramiPro** [comment]
dirty hack but it's work like charm 😁

**Pull request closed without merging**